packages/apollo-gateway/src/__tests__/buildQueryPlan.test.ts

```diff
@@ -93,69 +93,69 @@ describe('buildQueryPlan', () => {
     const queryPlan = buildQueryPlan(buildOperationContext(schema, query));
 
     expect(queryPlan).toMatchInlineSnapshot(`
-                                          QueryPlan {
```

Comment:
aw man this indentation change is a bummer.

(tests are not a bummer, tests are great, i'm just sad abt that jest bug)

Reply:
Good news is the bug is behind us and our poorly indented tests will resolve themselves over time 😄